src/main/webapp/queue/ta.html

```diff
@@ -10,45 +10,43 @@
   </head>
 
   <script>
```

Comment:
Do you think it would be clearer to put this script in a seperate js file?

Reply:
It's easier for me to have everything in the same file for now, but I'll separate everything out once I'm done with this feature!